lib/vmtranslator/translator.go

```diff
@@ -0,0 +1,141 @@
+package vmtranslator
+
+import (
+	"log"
+	"strconv"
+	"strings"
+)
+
+type AC_TYPE int
```

Comment:
whats with all these types? are they used for something?

Reply:
no ill get rid of them